routes/web.php

```diff
@@ -1,24 +1,24 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
-
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| contains the "web" middleware group. Now create something great!
-|
-*/
+use \App\Http\Controllers\PostController;
 
 Route::get('/', function () {
-    return view('welcome');
+    return redirect('/register');
 });
 
+Route::get('/register', function () {
+    return view('register');
+})->middleware(['auth'])->name('register');
+
 Route::get('/dashboard', function () {
     return view('dashboard');
 })->middleware(['auth'])->name('dashboard');
 
+Route::controller(PostController::class)->group(function () {
+    Route::get('/createPost', 'createPost');
+    Route::post('/post/create/{user_id}', 'create');
+    Route::get('/posts', 'posts');
+});
```

Comment:
These routes are intended for an authenticated user, so it would be better to wrap these routes in an `auth` middleware group.
```suggestion
Route::controller(PostController::class)->middleware('auth')->group(function () {
    Route::get('/createPost', 'createPost');
    Route::post('/post/create/{user_id}', 'create');
    Route::get('/posts', 'posts');
});
```